bids/layout/layout.py

```diff
@@ -14,6 +14,9 @@
 from .validation import BIDSValidator
 from .. import config as cf
 
+if six.PY2:
+    # To be able to pass encoding kwarg
+    from io import open
```

Comment:
I think this is a valid import in Python 3, as well, so no need to guard in the `if` block.

Reply:
I have some vague recollection that it is not "the same thing" but I was wrong:
```shell
$> python3 -c 'import io; print(io.open is open)' 
True
```
so indeed could have been imported unconditionally. But having this `PY3` could help later on to clean the py2 compatibility crude (ref #370) via simple grep ;-)